ArduCopter/mode.cpp

```diff
@@ -262,3 +261,16 @@ bool Copter::Mode::takeoff_triggered(const float target_climb_rate) const
 #endif
     return true;
 }
+
+void Copter::Mode::zero_throttle_and_relax_ac()
+{
+#if FRAME_CONFIG == HELI_FRAME  // Helicopters always stabilize roll/pitch/yaw
+        // call attitude controller
+        attitude_control->input_euler_angle_roll_pitch_euler_rate_yaw(0, 0, 0, get_smoothing_gain());
+        attitude_control->set_throttle_out(0,false,g.throttle_filt);
```

Comment:
Nitpicking (but I'm a bit tired of PRs fixing it later): use `0.0f` and add spaces between function arguments. This applies to the two above and another below.

I know you copied this from previous code but since we are moving it, we might as well fix it already 🙂

Reply:
You forgot to tell me to fix the indentation :-P